reference/5.1/Microsoft.PowerShell.Core/About/about_Jobs.md

```diff
@@ -67,116 +80,95 @@ To start a background job on the local computer, use the `Start-Job` cmdlet.
 To write a `Start-Job` command, enclose the command that the job runs in curly
 braces (`{}`). Use the **ScriptBlock** parameter to specify the command.
 
-The following command starts a background job that runs a `Get-Process`
-command on the local computer.
+The following command starts a background job that runs a `Get-Process` command
+on the local computer.
 
 ```powershell
 Start-Job -ScriptBlock {Get-Process}
 ```
 
+When you start a background job, the command prompt returns immediately, even
+if the job takes an extended time to complete. You can continue to work in the
+session without interruption while the job runs.
+
 The `Start-Job` command returns an object that represents the job. The job
 object contains useful information about the job, but it does not contain the
 job results.
 
-Save the job object in a variable, and then use it with the other Job cmdlets
-to manage the background job. The following command starts a job object and
-saves the resulting job object in the `$job` variable.
+Save the job object in a variable, and then use it with the other **Job**
```

Comment:
Can we reword the first sentence here?  It feels like a directive, and not an explanation.

Reply:
fixed